js/src/common/helpers/getFieldsAsArray.js

```diff
@@ -24,10 +24,6 @@ const compare = ( a, b ) => {
 const getFieldsAsArray = ( fields ) => {
 	return Object.keys( fields )
 		.reduce( ( accumulator, fieldName ) => {
-			if ( '' === fieldName ) {
-				return accumulator;
```

Comment:
It's possible for a field to not have a name, like if a user deletes the name (slug) before re-entering it. 

They should still see the field, even though it has no name.

Reply:
Might have to look into regressions this could cause in the normal block editor.